WordPress/Classes/ViewRelated/Stats/Insights/SiteStatsInsightsTableViewController.swift

```diff
@@ -572,6 +573,14 @@ extension SiteStatsInsightsTableViewController: SiteStatsInsightsDelegate {
         viewNeedsUpdating = false
     }
 
+    func scrollToNewCard() {
+        let lastSection = max(tableView.numberOfSections - 1, 0)
+
+        // newly added card will be penultimate row
+        let idxNewCardRow = max(tableView.numberOfRows(inSection: lastSection) - 2, 0)
+        tableView.scrollToRow(at: IndexPath(row: idxNewCardRow, section: lastSection), at: .bottom, animated: true)
```

Comment:
I think `.middle` looks slightly better here, as we then get some padding underneath the Add Stats Card row at the bottom (otherwise it sits on top of the tab bar).